engines/wintermute/base/base_keyboard_state.cpp

```diff
@@ -432,6 +576,82 @@ uint32 BaseKeyboardState::keyCodeToVKey(Common::Event *event) {
 
 //////////////////////////////////////////////////////////////////////////
 Common::KeyCode BaseKeyboardState::vKeyToKeyCode(uint32 vkey) {
+#ifdef ENABLE_FOXTAIL
```

Comment:
This might get cleaner if you create a separate function for the foxTail-specifics, and call that from here, that way this function avoids becoming this much longer.

Reply:
Done in fd3846e. Also changed constant names according to http://wiki.scummvm.org/index.php/Code_Formatting_Conventions#Naming